packages/workspace/src/migrations/update-13-2-0/set-parallel-default.ts

```diff
@@ -4,18 +4,19 @@ import {
   Tree,
   updateWorkspaceConfiguration,
 } from '@nrwl/devkit';
-import { output } from '../../utilities/output';
 
 export async function setParallelDefault(host: Tree) {
   const config = readWorkspaceConfiguration(host);
-  if (config.tasksRunnerOptions['default'].options.parallel) {
-    config.tasksRunnerOptions['default'].options.parallel =
-      config.tasksRunnerOptions['default'].options.maxParallel || 3;
-    delete config.tasksRunnerOptions['default'].options.maxParallel;
-  } else {
-    config.tasksRunnerOptions['default'].options.parallel = 1;
+  if (config.tasksRunnerOptions && config.tasksRunnerOptions['default']) {
```

Comment:
Nit: use optional chaining instead of &&

Reply:
Good suggestion, updated now 👍